### ISSUE TITLE
Destroying listeners

### ISSUE THREAD

**Issue by einarnordengren** [d954mas/defold-box2d#20]
Is there any way to destroy contact listeners or destruction listeners? It looks like the references to them are destroyed in the world's Destroy() function, but the number of LuaRefs keeps increasing every time I reinitalize the world.

**Comment by d954mas** [comment]
If you set nil, Listener should be destroyed

https://github.com/d954mas/defold-box2d/blob/bb6ca239b14b7350e6ff9b0ca223dcf42c48e926/box2d/native/src/world.cpp#L67

**Comment by d954mas** [comment]
I look at code. It looks like I forgot to remove listeners when destroy world.

https://github.com/d954mas/defold-box2d/blob/bb6ca239b14b7350e6ff9b0ca223dcf42c48e926/box2d/native/src/world.cpp#L41

**Comment by einarnordengren** [comment]
Right, so it should work by just destroying the world. However, if I add
```
self.world:SetContactListener({
    BeginContact = function() end,
    EndContact = function() end,
    PreSolve = function() end,
    PostSolve = function() end
})
``` 
in the init function of the dominos_scene.script, the number LuaRefs increases (checked in the web profiler) every time I press restart even if the world is destroyed in final.
